schema/archive.json

```diff
@@ -0,0 +1,210 @@
+{
+  "collections": {
+    "archives": {
+      "name": "archives",
+      "config": {
+        "displayFields": [
+          "id", "target.product", "target.channel", "target.version",
+          "target.platform", "target.locale"
+        ],
+        "schema": {
+          "title": "Archives",
+          "description": "Mozilla software builds.",
+          "type": "object",
+          "additionalProperties": false,
+          "required": [
+            "id",
+            "source",
+            "download",
+            "target"
+          ],
+          "properties": {
+            "id": {
+              "type": "string",
+              "title": "ID",
+              "description": "Build Hub ID"
+            },
+            "schema": {
+              "type": "integer",
+              "title": "Schema version",
+              "description": "Schema version"
+            },
```

Comment:
I know for sure that last_modified is stripped by the admin. Probably these two are too.